modules/tensor_mechanics/doc/content/source/materials/crystal_plasticity/ComputeMultipleCrystalPlasticityStress.md

```diff
@@ -196,7 +196,74 @@ The inverse deformation gradient is used to relate
 the current configuration frame back to the reference frame (i.e., pull back). Note in the `ComputeMultipleCrystalPlasticityStress` class,
 the pull back of Cauchy stress is via the elastic part of the deformation gradient ($\boldsymbol{F}^e$) only (see [eqn:stress_cauchy]).
 
-## Computation workflow
+### Calculation of Schmid Tensor
+
+The calculation of the flow direction Schmid tensor, the dyadic product of the slip direction and slip plane normal unit vectors, $\boldsymbol{s}_{i,o}^{\alpha} \otimes \boldsymbol{m}_{i,o}^{\alpha}$, is straight forward for the case of cubic crystals, including Face Centered Cubic (FCC) and Body Centered Cubic (BCC) crystals. The 3-index Miller indices commonly used to describe the slip direction and slip plane normals are first normalized individually normalized and then directly used in the dyadic product.
+
+#### Conversion of Miller-Bravis Indices for HCP to Cartesian System
+
+Hexagonal Close Packed (HCP) crystals are often described with the 4-index Miller Bravis system:
+\begin{equation}
+  \label{eqn:millerBravisHCPIndices}
+  (HKIL) [uvtw]
+\end{equation}
+To compute the Schmid tensor from these slip direction and slip plane normals, the indices must first be transformed to the Cartesian coordinate system. Within the associated `ComputeMultipleCrystalPlasticityStress` implementation, this conversion uses the assumption that the a$_1$-axis, or the H index, align with the x-axis in the basal plane of the HCP crystal lattice. The c-axis, the L index, is assumed to be paralled to the z-axis of the Cartesian system.
+
```

Comment:
This is probably too much to ask - do you think it would be better to insert a diagram showing the axis and indices (I remember seeing something like this in your presentations before?)

Reply:
I had copied from a paper, rather than draw my own figure, but it is a good idea to have the image to clarify the convention. thanks!